### DISCUSSION TITLE
Linux-Command

### DISCUSSION THREAD

**Discussion by JSHT** [1101-GS4719-C/Python-Community-GS4719-C#668] [優秀專案]
# [linux-command](https://github.com/jaywcjlove/linux-command)
大家耳熟能詳的三大作業系統 Windows、MacOS、Linux。其中 Linux 作為開源的作業系統，目前有各式各樣的版本被開發出來，例如 Linux Mint、manjaro、MX Linux 等。在接觸、學習 Linux，需要習慣使用指令的方式進行操作。其中除了常使用的指令，也有各種極為方便的指令，這個 project 就是讓大家能夠快速查詢、上手的工具。

**Comment by treeann** [comment]
原來linux的指令這麼多用途

**Comment by sys511613** [comment]
這個很方便耶，之後如果要用linux有關的東西都可以參考這個，感謝分享!

**Comment by jcpan821** [comment]
蠻有意思的

**Comment by stan890425** [comment]
感謝分享~以後用到linux就可以查了

**Comment by alexthebigpp** [comment]
想學linux很久了，有這個感覺可以更快上手

**Comment by minniewei** [comment]
感謝分享呀呀呀，linux真的很重要

**Comment by yenwei12138** [comment]
以前家裡的電腦就是Linux系統，一直覺得很陌生不太會用，真可惜沒有早點知道這玩意

**Comment by Ben6uo** [comment]
一直覺得Linux系統遲早會用很廣 真的太神啦

**Comment by Zoe06060606** [comment]
之前在學Linux 超痛苦的🥲
都看不懂也不不知道怎麼查
感謝分享這個整理庫！
